NEWS.md

```diff
@@ -4,6 +4,18 @@ ClimaAtmos.jl Release Notes
 Main
 -------
 
+v0.27.2
+-------
+- Implements TOA edmf model
+  PR [#3215](https://github.com/CliMA/ClimaAtmos.jl/pull/3215)
+- Use new aerosol artifact and change start date
+  PR [#3216](https://github.com/CliMA/ClimaAtmos.jl/pull/3216)
+- Add a gpu scaling job with diagnostics
+  PR [#2852](https://github.com/CliMA/ClimaAtmos.jl/pull/2852)
+- Change dt
+  PR [#3213](https://github.com/CliMA/ClimaAtmos.jl/pull/3213)
+  PR [#3212](https://github.com/CliMA/ClimaAtmos.jl/pull/3212)
```

Comment:
These ones, too. They are "maintenance"

Reply:
thanks! I wasn't sure if I should put it...